dlp/src/main/java/com/example/dlp/Templates.java

```diff
@@ -0,0 +1,257 @@
+   * @param infoTypeList The infoTypes of information to match
+   * @param minLikelihood The minimum likelihood required before returning a match
+   * @param maxFindings The maximum number of findings to report per request (0 = server maximum)
+   */
+  private static void createInspectTemplate(
+      String displayName,
+      String templateId,
+      String description,
+      String projectId,
+      List<InfoType> infoTypeList,
+      Likelihood minLikelihood,
+      int maxFindings) {
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+
+      FindingLimits findingLimits =
+          FindingLimits.newBuilder().setMaxFindingsPerRequest(maxFindings).build();
+
+      // Construct the inspection configuration for the template
+      InspectConfig inspectConfig =
+          InspectConfig.newBuilder()
+              .addAllInfoTypes(infoTypeList)
+              .setMinLikelihood(minLikelihood)
+              .setLimits(findingLimits)
+              .build();
+
+      InspectTemplate inspectTemplate =
+          InspectTemplate.newBuilder()
+              .setInspectConfig(inspectConfig)
+              .setDisplayName(displayName)
+              .setDescription(description)
+              .build();
+
+      CreateInspectTemplateRequest createInspectTemplateRequest =
+          CreateInspectTemplateRequest.newBuilder()
+              .setParent(ProjectName.of(projectId).toString())
+              .setInspectTemplate(inspectTemplate)
+              .setTemplateId(templateId)
+              .build();
+
+      InspectTemplate response =
+          dlpServiceClient.createInspectTemplate(createInspectTemplateRequest);
+      System.out.printf("Template created: %s", response.getName());
+    } catch (Exception e) {
+      System.out.printf("Error creating template: %s", e.getMessage());
+    }
+  }
+  // [END dlp_create_template]
+
+  /**
+   * [START dlp_list_templates]
+   *
+   * <p>List DLP inspection templates created in a given project
+   *
+   * @param projectId Google Cloud Project ID
+   */
+  private static void listInspectTemplates(String projectId) {
+
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+
+      ListInspectTemplatesRequest request =
+          ListInspectTemplatesRequest.newBuilder()
+              .setParent(ProjectName.of(projectId).toString())
+              .setPageSize(1).build();
+
+      ListInspectTemplatesPagedResponse response = dlpServiceClient.listInspectTemplates(request);
+      ListInspectTemplatesPage page = response.getPage();
+      ListInspectTemplatesResponse templatesResponse = page.getResponse();
+
+      for (InspectTemplate template : templatesResponse.getInspectTemplatesList()) {
+        System.out.printf("Template name: %s\n", template.getName());
+        if (template.getDisplayName() != null) {
+          System.out.printf("Template display name: %s \n", template.getDisplayName());
```

Comment:
Nit 1: add a `\t` in front of the template property print-outs?
Nit 2: remove `template` from everything but the `Template name` one?

Reply:
Done.